src/components/EmailSubscription/index.view.tsx

```diff
@@ -1,9 +1,67 @@
-import React from "react";
+import React, { useState } from "react";
+import axios, { AxiosResponse } from "axios";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+
 import "./EmailSubscription.scss";
 
 const EmailSubscriptionForm: React.FC = () => {
+  const [userEmail, setUserEmail] = useState("");
+  const [feedbackMessage, setFeedbackMessage] = useState("");
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  const handleInput = (event: React.FormEvent<HTMLInputElement>) => {
+    setUserEmail(event.currentTarget.value.trim());
+  };
+
+  const handleKey = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "Enter") {
+      submitEmail();
+    }
+  };
+
+  const submitEmail = () => {
+    axios
+      .post(
+        "https://us-central1-cruzhacks-4a899.cloudfunctions.net/subscribe",
+        { email: userEmail }
+      )
+      .then((response: AxiosResponse) => {
+        setShowFeedback(true);
+        setFeedbackMessage(response.data.message);
+      })
+      .catch(() => {
+        setShowFeedback(true);
+        setFeedbackMessage(
+          "Oh no! Your submission looks like it failed - please try your request again!"
```

Comment:
this error message may be too long— we will need to think about wrapping this text on mobile

Reply:
Bump in this @tim-nguyen-cs

Reply:
Take a look at how I hard-coded the widths on my current revision (pushed) and current iteration (not yet pushed), and let me know what else we should do:
```
  &__feedbackMessage {
    @include typography.description;
    color: var(--wedgewood);
    width: 350px;
    margin-top: var(--extra-small-spacer);

    @include responsiveness.respond(mission) {
      width: min(48vw, 300px);
    }
  }
```

In short, it wraps relative to the search box.